.github/workflows/embedded_ui_refresh.yaml

```diff
@@ -0,0 +1,71 @@
+on:
+  repository_dispatch:
+    types: [embedded_ui_refresh]
+  workflow_dispatch:
+    inputs:
+      asset_name:
+        description: Release asset name
+        required: true
+        default: embedded-ui
+      repository:
+        description: Source repository
+        required: true
+        default: ydb-platform/ydb-embedded-ui
+      tag_name:
+        description: Release tag
+        required: true
+
+name: Embedded UI Refresh
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      ASSET_NAME: ${{ github.event.inputs && github.event.inputs.asset_name || github.event.client_payload.asset_name }}
+      REPOSITORY: ${{ github.event.inputs && github.event.inputs.repository || github.event.client_payload.repository }}
```

Comment:
Will values form `client_payload` will ever be applied, while `inputs.asset_name` and `inputs.repository` have default values?